docs/fast_data/bucket_storage_support/configuration/ingestion_storer.md

```diff
@@ -4,22 +4,15 @@ title: Ingestion Storer Configuration
 sidebar_label: Ingestion Storer
 ---
 
-In this page it will be described how to configure the Ingestion Storer service, which is the one responsible
-for consuming messages from a messages streaming platform, in particular Apache Kafka, and storing them into a Cloud Storage Bucket.
+This page describes how to configure the Ingestion Storer service which is responsible for consuming messages from a message streaming platform and storing them into a Cloud Storage Bucket.
 
 ## Overview
 
-Ingestion Storer service reads from the streaming platform the ingestion messages through the poll action, which returns
-all the available messages from subscribed _ingestion_ topics. Then those messages are grouped by their
-topic name and partition to be written on the bucket. In this manner each stored file contains
-only messages coming from a single partition, so that multiple service replicas do not interfere
-with each other. Moreover, this saving logic simplifies potential messages reorganization.
+Ingestion Storer service reads the ingestion messages from the chosen message streaming platform. Then those messages are grouped by topic name and partition so they can later be saved in the bucket. This way each stored file contains only messages coming from a single partition, so that multiple service replicas do not interfere with each other. Moreover, this saving logic simplifies potential messages reorganization.
```

Comment:
```suggestion
The Ingestion Storer service reads the ingestion messages from the chosen message streaming platform. Then those messages are grouped by topic name and partition so they can later be saved in the bucket. This way each stored file contains only messages coming from a single partition, so that multiple service replicas do not interfere with each other. Moreover, this saving logic simplifies potential messages reorganization.
```